core/driver/tests/src/test/scala/org/virtuslab/ideprobe/ProbeDriverTest.scala

```diff
@@ -185,6 +185,32 @@ final class ProbeDriverTest extends IdeProbeFixture with Assertions with RobotPl
     }
   }
 
+  @Test
+  def goToLineTest(): Unit = {
+    buildTestFixture.run { intelliJ =>
+      val projectDir = intelliJ.workspace.resolve("simple-sbt-project")
+      intelliJ.probe.withRobot.openProject(projectDir)
+
+      val file = projectDir.resolve("src/main/scala/Highlighting.scala")
+      file.write("""package wrong
+                   |
+                   |class NonCorrespondingName {
+                   |  val immutable = 4
+                   |  immutable = 3
+                   |
+                   |  List(1, 2, 3).exists(_ == 3)
+                   |}
+                   |""".stripMargin)
+      intelliJ.probe.syncFiles()
+
+      intelliJ.probe.openEditor(file)
+      intelliJ.probe.goToLineColumn(ProjectRef.Default, 5, 4)
```

Comment:
Strange that this api was not fixed yet. Could you make it match other endpoints? 

def goToLineColumn(line: Int, column: Int, projectRef: ProjectRef = ProjectRef.Default):